### PR TITLE
[EM-559] Add Ezypay to Android sdk

### PR DESCRIPTION
**Pull request by Ashraf-alkibsi**
Add Ezypay installment to the sdk

### CODE REVIEW

File: sdk/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -65,6 +65,7 @@
     <string name="payment_method_credit_card_title">クレジットカード</string>
     <string name="payment_method_installment_bay_title">クルンシィ</string>
     <string name="payment_method_installment_bbl_title">バンコク銀行</string>
+    <string name="payment_method_installment_ezypay_title">Maybank Ezypay</string>
```
Comment (nuxzero) [inline sdk/src/main/res/values-ja/strings.xml:68]: Could you translate this to Japan?

File: sdk/src/main/res/values-th/strings.xml [hunk 2]
```diff
@@ -65,6 +65,7 @@
     <string name="payment_method_credit_card_title">บัตรเครดิต</string>
     <string name="payment_method_installment_bay_title">ธนาคารกรุงศรี</string>
     <string name="payment_method_installment_bbl_title">ธนาคารกรุงเทพ</string>
+    <string name="payment_method_installment_ezypay_title">Maybank Ezypay</string>
```
Comment (nuxzero) [inline sdk/src/main/res/values-th/strings.xml:68]: Also this one.

**Review by danfowler — APPROVED** [review]

**Review by kan98 — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

File: sdk/src/main/res/values-ja/strings.xml (hunk 1 above)
Comment (Ashraf-alkibsi) [reply sdk/src/main/res/values-ja/strings.xml:68]: I am afraid that we don't have the Japanese translation.

Comment (nuxzero) [reply sdk/src/main/res/values-ja/strings.xml:68]: I think translation team could help you translate this.

Comment (Ashraf-alkibsi) [reply sdk/src/main/res/values-ja/strings.xml]: thanks to @hpariwat I got a translation.

**Review by nuxzero — APPROVED** [review]

**Pull request merged**